lib/spack/docs/tutorial_modules.rst

```diff
@@ -776,113 +1103,169 @@ After modifications the configuration file will be:
             OMPI_MCA_btl_openib_warn_default_gid_prefix: '0'
 
 
-.. note::
-  The double colon
-    The double colon after ``enable`` is intentional and it serves the
-    purpose of overriding the default list of enabled generators so
-    that only ``lmod`` will be active (see :ref:`the reference
-    manual <config-overrides>` for a more detailed explanation of
-    config scopes). If a single colon is used, it will append instead
-    of override.
-
-The directive ``core_compilers`` accepts a list of compilers; everything built
-using these compilers will create a module in the ``Core`` part of the hierarchy. It is
+The double colon after ``enable`` is intentional and it serves the
```

Comment:
I liked the fact that this was called out in a note, it's a subtle point and worth highlighting!